Tribler/Core/CacheDB/SqliteCacheDBHandler.py

```diff
@@ -1525,6 +1525,17 @@ def on_remove_torrent_from_dispersy(self, channel_id, dispersy_id, redo):
 
         self.notifier.notify(NTFY_CHANNELCAST, NTFY_UPDATE, channel_id)
 
+        sql = """SELECT infohash, dispersy_cid FROM Torrent, _ChannelTorrents, Channels
+        WHERE Torrent.torrent_id = _ChannelTorrents.torrent_id
+        AND _ChannelTorrents.channel_id = ? AND _ChannelTorrents.dispersy_id = ?
+        AND Channels.id = _ChannelTorrents.channel_id"""
+        infohash, dispersy_cid = self._db.fetchone(sql, (channel_id, dispersy_id))
+
+        if infohash:
+            self.notifier.notify(NTFY_TORRENTS, NTFY_DELETE, None,
```

Comment:
For consistency I looked at other ``NTFY_TORRENTS`` events and they _all_ have infohash instead of None here.

I argue this torrent discovered event is the odd one here.